pkg/webhooks/clusterrolebinding/clusterrolebinding.go

```diff
@@ -0,0 +1,200 @@
+package clusterrolebinding
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"regexp"
+
+	"github.com/openshift/managed-cluster-validating-webhooks/pkg/webhooks/utils"
+	admissionv1 "k8s.io/api/admission/v1"
+	admissionregv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	admissionctl "sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookName       string = "clusterrolebindings-validation"
+	docString         string = `Managed OpenShift Customers may not delete the cluster role bindings under the managed namespaces: %s`
+	managedNamespaces string = `(^openshift-.*|kube-system)`
+)
+
+var (
+	timeout int32 = 2
+	log           = logf.Log.WithName(WebhookName)
+	scope         = admissionregv1.ClusterScope
+	rules         = []admissionregv1.RuleWithOperations{
+		{
+			Operations: []admissionregv1.OperationType{"DELETE"},
+			Rule: admissionregv1.Rule{
+				APIGroups:   []string{"rbac.authorization.k8s.io"},
+				APIVersions: []string{"v1"},
+				Resources:   []string{"clusterrolebindings"},
+				Scope:       &scope,
+			},
+		},
+	}
+
+	protectedNamespaces = regexp.MustCompile(managedNamespaces)
+
+	exceptionNamespaces = []string{
```

Comment:
Do we still need to validate the namespace if it is ClusterScope?

Reply:
yeh, `clusterrolebindings` links to the subject  `ServiceAccount` which always comes through a namespace. 
basically, this var allows modifying `clusterrolebindings` which links `exception-Namespaces`